documentation/source/conf.py

```diff
@@ -68,11 +68,15 @@
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
-#
+
+version_raw = open(os.path.join("..", "..", "version")).read()[8:].strip()
```

Comment:
```suggestion
version_raw = open(os.path.join(os.path.dirname(__file__), "..", "..", "version")).read()[8:].strip()
```
Would still prefer the solution I propose in the comment above

Reply:
Done that for now.